src/main/java/org/openmbee/mdk/mms/sync/delta/DeltaSyncRunner.java

```diff
@@ -78,9 +83,13 @@ public void run(ProgressStatus progressStatus) {
             Utils.guilog("[ERROR] An error occurred while validating credentials. Credentials will be cleared. Skipping sync. All changes will be persisted in the model and re-attempted in the next sync. Reason: " + e.getMessage());
             return;
         }
+        progressStatus.increase();
+        progressStatus.setDescription("Coordinated Sync: Project Validation");
+        progressStatus.cancelIfCanceled();
 
         ProjectValidator pv = new ProjectValidator(project);
-        pv.validate();
+        ProgressStatusRunner.runWithProgressStatus(pv, "Coordinated Sync: Project Validation", true, 0);
```

Comment:
is the reason why it's starting a new runner instead of calling pv.run(progressStatus) because it's changing to indeterminate?